css/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.6rem;
+  color: var(--secondary-color);
+}
+
+:root {
+  --primary-color: #6070ff;
+  --secondary-color: #344563;
+  --third-color: #396df2;
+  --abstract-color: #172b4d;
+  --simple-color: #ebebff;
+  --neutral-color: #7a869a;
+  --normal-color: #000;
+  --general-color: #f7f7f9;
+  --sample-color: #7f8cff;
+}
+
+body {
+  background-color: var(--general-color);
+}
```

Comment:
[OPTIONAL]
Since you were considered working only on the mobile version it can be good to delete all this section on top of the `media`